pkg/common/plugin/exec.go

```diff
@@ -102,17 +103,16 @@ func (e *DefaultExecutor) ExecutePlugin(ctx context.Context, pluginPath string,
 }
 
 func (e *DefaultExecutor) pluginErr(err error, stdout, stderr []byte) error {
-	errMsg := Error{}
-	if len(stdout) == 0 {
-		if len(stderr) == 0 {
-			errMsg.Msg = fmt.Sprintf("plugin failed with no proper error message: %v", err)
-		} else {
-			errMsg.Msg = fmt.Sprintf("plugin failed with error: %q", string(stderr))
-		}
-	} else if perr := json.Unmarshal(stdout, &errMsg); perr != nil {
-		errMsg.Msg = fmt.Sprintf("plugin failed and parsing its error message also failed with error %q: %v", string(stdout), perr)
-	}
-	return &errMsg
+	var stdOutBuffer bytes.Buffer
+	var stdErrBuffer bytes.Buffer
+
+	// writing them to buffer to avoid lint error
+	stdOutBuffer.Write(stdout)
+	stdErrBuffer.Write(stderr)
```

Comment:
just curious what's difference between `string(stderr)` and `stdErrBuffer.Write(stderr)`

Reply:
there are no underlaying differences, however lint doesn't like string()